src/libs/ReportUtils.ts

```diff
@@ -4437,7 +4437,7 @@ function goBackToDetailsPage(report: OnyxEntry<Report>, backTo?: string) {
     }
 }
 
-function navigateBackOnDeleteTransaction(backRoute: Route | undefined, isFromRHP?: boolean) {
+function navigateBackOnDeleteTransaction(backRoute: Route | undefined, isFromRHP?: boolean, reportID?: string) {
```

Comment:
What reportID is being passed here? Is it the reportID that the transaction itself is on?